llama.cpp

```diff
@@ -2629,7 +2657,23 @@ struct llama_context * llama_init_from_file(
     return ctx;
 }
 
+struct llama_context * llama_init_from_file(
```

Comment:
I vote to remove this. I prefer giving user one choice only, which is:
model = load_model
context = new_context_from_model.

Giving second choice of create context from model file name is not worth the code and confusioni.

Reply:
#ifdef __GNUC__
#define DEPRECATED(func) func __attribute__ ((deprecated))
#elif defined(_MSC_VER)
#define DEPRECATED(func) __declspec(deprecated) func
#else
DEPRECATED(func) func
#endif